app/modules/asset_groups/models.py

```diff
@@ -68,9 +68,11 @@ def __enter__(self):
             self.repo.remotes.origin.set_url(self.authenticated_url)
         return self
 
-    def __exit__(self, type, value, traceback):
+    def __exit__(self, exc_type, exc_value, traceback):
         if self.repo is not None:
             self.repo.remotes.origin.set_url(self.original_url)
+        if exc_type:
```

Comment:
what's "exc" mean in this context? It's not clear to me

Reply:
`exc` is exception.  This is the documentation https://docs.python.org/3/library/stdtypes.html?highlight=__exit__#contextmanager.__exit__

We were (unintentionally?) catching all exceptions inside `with GitLabPAT`